build-scripts/packageExtension.js

```diff
@@ -0,0 +1,118 @@
+/*!
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: MIT
+ */
+
+const fs = require('fs-extra')
+const path = require('path')
+const ncp = require('child_process')
+const shell = require('shelljs')
+
+const timeMessage = 'Packaged extension'
+const manifestFile = 'vss-extension.json'
+const tasksDirectory = 'Tasks'
+const repoRoot = path.dirname(__dirname)
+const inTasks = path.join(repoRoot, tasksDirectory)
+const outBuildTasks = path.join(repoRoot, '_build', tasksDirectory)
+const outPackage = path.join(repoRoot, '_package')
+const outPackageTasks = path.join(outPackage, tasksDirectory)
+
+const unprocessFolders = [
```

Comment:
consider `ignore` as a more standard terminology (eg .gitignore/.dockerignore)

Reply:
sure, changed all of these lists to ingore